night_of_the_dead/egg-night-of-the-dead.json

```diff
@@ -0,0 +1,200 @@
+{
+    "_comment": "DO NOT EDIT: FILE GENERATED AUTOMATICALLY BY PANEL",
+    "meta": {
+        "version": "PTDL_v2",
+        "update_url": null
+    },
+    "exported_at": "2024-06-12T09:42:36+00:00",
+    "name": "Night Of The Dead",
+    "author": "marshals@gmail.com",
+    "uuid": "08b518e7-fa75-4db9-9fdc-07fc9bc35c28",
+    "description": "Build defenses against nightly waves of zombies. Survive and escape the island.",
+    "features": [
+        "steam_disk_space"
+    ],
+    "docker_images": {
+        "wine_latest": "ghcr.io\/parkervcp\/yolks:wine_latest"
+    },
+    "file_denylist": [],
+    "startup": "xvfb-run --auto-servernum wine \/home\/container\/LF\/Binaries\/Win64\/LFServer-Win64-Shipping.exe ?listen -Port={{SERVER_PORT}} -QueryPort={{QUERY_PORT}} -console -log --CRASHREPORTS {{OTHER_ARGS}}",
+    "config": {
+        "files": "{\n    \".\/LF\/Saved\/Config\/ServerSettings.ini\": {\n        \"parser\": \"ini\",\n        \"find\": {\n            \"ServerSettings.ServerName\": \"{{server.environment.SERVER_NAME}}\",\n            \"ServerSettings.Password\": \"{{server.environment.SERVER_PASSWORD}}\",\n            \"SystemSettings.SaveName\": \"\/home\/container\/Saved\/SaveGames\/{{server.environment.SERVER_SAVEGAME}}\",\n            \"SystemSettings.IngameAdminPassword\": \"{{server.environment.SERVER_ADMIN_PASSWORD}}\",\n            \"SystemSettings.DefaultMessageOfTheDay\": \"{{server.environment.SERVER_MOTD}}\",\n            \"GameSettings.Difficulty\": \"{{server.environment.DIFFICULTY}}\",\n            \"GameSettings\/General.MODSelect\": \"{{server.environment.PVP}}\"\n        }\n    }\n}",
```

Comment:
SystemSettings.SaveName should have a path change  including the 'LF' directory in the path. /home/container/LF/Saved/SaveGames/{{server.environment.SERVER_SAVEGAME}}